static/js/components/LearningResourceCard.js

```diff
@@ -55,11 +59,45 @@ const getPlatform = (object: LearningResourceSummary): string => {
   }
 }
 
+const getPlatformName = object => platformReadableNames[getPlatform(object)]
+
+const getClassName = searchResultLayout =>
+  `learning-resource-card ${
+    searchResultLayout === SEARCH_LIST_UI ? "list-view" : ""
+  }`.trim()
+
+const formatTopics = (topics: Array<{ name: string }>) =>
+  topics.map(topic => topic.name).join("  ")
```

Comment:
Since browsers collapse consecutive whitespace into one:
```suggestion
  topics.map(topic => topic.name).join("\u00A0\u00A0")
```